docs/execute/binaries.md

```diff
@@ -36,6 +36,11 @@ But, obviously, testing validated versions of components is not really interesti
     * `DataDog/dd-trace-js#master`
 2. Clone the dd-trace-js repo inside `binaries`
 
+OR
+
+1. Create a file `nodejs-load-from-load` in `binaries/`, the content will be installed by `npm link` to the local folder
```

Comment:
```suggestion
3. Create a file `nodejs-load-from-load` in `binaries/`, the content will be installed by `npm link` to the local folder
```

Reply:
also `nodejs-load-from-what???`

Reply:
This step is not a step that comes after 2, it's a step that replaces 2.

> also nodejs-load-from-what???

Ah yes it should be `from-local`, will fix.

Reply:
Fixed the incorrect name, but I disagree with the change as it's not an additional step, it replaces the previous steps entirely.

Reply:
But Roch I don't think they're steps ? I think they're already alternatives no ?

Reply:
just like in parametric.md btw, same principle

Reply:
we should just copy those two files tbh:
```markdown
3. Create a file `nodejs-load-from-local` in `binaries/`, this will disable installing with `npm install dd-trace` and
   will instead get the content of the file, and use it as a location of the `dd-trace-js` repo and then mount it as a
   volume and `npm link` to it. For instance, if this repo is at the location, you can set the content of this file to
   `../dd-trace-js`. This also removes the need to rebuild the weblog image since the code is mounted at runtime.
```

Reply:
Oh right, I misread that entirely. Will change it.